File: piyunggur/app/src/main/java/com/example/user/ramen/LogoActivity.java
package com.example.user.ramen;

import android.content.Intent;
import android.os.CountDownTimer;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;

public class LogoActivity extends AppCompatActivity {

    private Button bNext;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.logo_layout);

        new CountDownTimer(1500,1) {

            public void onTick(long millisUntilFinished) {

            }

            public void onFinish() {
                startActivity(new Intent(LogoActivity.this,LearnActivity.class));
                finish();
            }
        }.start();

    }
}

File: RAMEN/app/src/main/java/com/example/piyunggur/ramen/LogoActivity.kt
package com.example.piyunggur.ramen

import android.content.Intent
import android.os.CountDownTimer
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class LogoActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.logo_layout)

//        object : CountDownTimer(1500, 1) {
//            override fun onTick(millisUntilFinished: Long) {
//
//            }
//
//            override fun onFinish() {
//                //sstartActivity(Intent(this@LogoActivity, LearnActivi ty::class.java))
//                finish()
//            }
//        }.start()
    }
}
